frontend/src/lib/components/apps/components/display/dbtable/AppDbExplorer.svelte

```diff
@@ -69,30 +72,26 @@
 			timeoutInput = undefined
 			console.log('compute input')
 			aggrid?.clearRows()
-			input = createPostgresInput(
+
+			input = getSelectInput(
 				resource,
-				resolvedConfig.type.configuration.postgresql.table,
+				resolvedConfig.type.configuration[resolvedConfig.type.selected].table,
 				columnDefs,
-				whereClause
+				whereClause,
+				resolvedConfig.type.selected as DbType
 			)
-		}, 1000)
+		}, 500)
 	}
 
 	$: editorContext != undefined && $mode == 'dnd' && resolvedConfig.type && listTableIfAvailable()
 
 	$: editorContext != undefined &&
 		$mode == 'dnd' &&
-		resolvedConfig.type.configuration?.postgresql?.table &&
+		resolvedConfig.type.configuration?.[resolvedConfig?.type?.selected]?.table &&
 		listColumnsIfAvailable()
 
-	let firstQuicksearch = true
-	$: if (quicksearch) {
-		if (firstQuicksearch) {
-			firstQuicksearch = false
-		} else {
-			aggrid?.clearRows()
-		}
-	}
+	// Everythime quicksearch changes, we want to clear the rows
+	$: quicksearch != undefined && aggrid?.clearRows()
```

Comment:
Why was this changed ? Not it clearRows() at initialization which can introduce race issues

Reply:
Fixed